src/components/forms/DateField/DateField.js

```diff
@@ -1,21 +1,20 @@
 import React, { useState } from 'react';
-import moment from 'moment';
 import FormControl from '@material-ui/core/FormControl';
 import { KeyboardDatePicker } from '@material-ui/pickers';
 
 import useStyles from './styles';
-import { useForm } from '../../FormProvider/FormProvider';
+import { useForm } from 'components/FormProvider';
 
-function DateField({ formKey, label }) {
+function DateField({ name, label, defaultValue }) {
   const styles = useStyles();
   const { setForm } = useForm();
-  const [selectedDate, setDate] = useState(moment());
+  const [selectedDate, setDate] = useState(defaultValue);
```

Comment:
Now that we're supporting `defaultValue`, shouldn't we also be setting it on the `form` here?  If the user chooses to accept the default value, then the `handleDateChange` function will never get called and the date will never actual be put in the `form` field.

Reply:
Good observation! For this and a few other annoying reasons, I think we should use a library called [Formik](https://jaredpalmer.com/formik/) which takes care of keeping track of values (including initial values), errors, submission, and at some point validation if we like. I noticed our components were getting a little unwieldy and thought I'd see how much work it would be to convert now rather than later (turns out not too much work). @radamson and I discussed this possibility so my next PR should fix this issue.